src/libutil/namespaces.cc

```diff
@@ -52,7 +52,7 @@ bool userNamespacesSupported()
 
             auto r = pid.wait();
             assert(!r);
-        } catch (SysError & e) {
+        } catch (SystemError & e) {
```

Comment:
These can only be POSIX errors so it's fine to keep them as `SysError`.

Reply:
Turned back!